### ISSUE TITLE
CA1001 Fixer generates incorrect code

### ISSUE THREAD

**Issue by vbjay** [dotnet/roslyn-analyzers#292]
![image](https://cloud.githubusercontent.com/assets/3629489/9752565/f91a693e-567f-11e5-9979-101d1cc04365.png)
As you can see.  When you run the fixer with a class that already implements an interface, it adds a 
```VB
, IDisposable
```
line to the code file.  The only problem is that the Implements statement does not allow multiple lines unless the "," is on the end of the line like this

```VB
 Implements IExporter,
            IDisposable
```

**Comment by vbjay** [comment]
See dotnet/roslyn#5097

**Comment by mattwar** [comment]
This is fixed in #5502 and will be available in next release of Roslyn.

**Comment by tmeschter** [comment]
Resolving as "External" as the issue was in the `SyntaxGenerator` type from the dotnet/roslyn repo. Code Fixes will pick this up automatically when running on the next version of Roslyn.
